pkg/cmd/pulumi/packagecmd/project_sdk_link.go

```diff
@@ -371,10 +371,15 @@ func linkPythonPackage(ws pkgWorkspace.Context, root string, pkg *schema.Package
 		importName = strings.ReplaceAll(pkg.Name, "-", "_")
 	}
 
+	namespace := "pulumi"
+	fmt.Println("namespace", pkg.Namespace)
```

Comment:
Do we want to print this?

Reply:
No, this existed in an intermediate commit, but not in the final diff.  Sorry for not cleaning up the history.